### ISSUE TITLE
secureCRT与figure冲突

### ISSUE THREAD

**Issue by yongqian0816** [withfig/fig#1629]
### Description:
> Please include a detailed description of the issue (and an image or screen recording, if applicable)

### Details:
|macOS|Fig|Shell|
|-|-|-|
|12.4.0|Version 1.0.58 (B472)|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Version 1.0.58 (B472) [ABC] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/qianyong/.fig/bin/fig
Accessibility: true
SSH Integration: false
Tmux Integration: false
iTerm Integration: false 
Hyper Integration: false
VSCode Integration: true
Docker Integration: true
Symlinked dotfiles: false
Only insert on tab: false
UNIX Socket Exists: true
Installation Script: true
PseudoTerminal Path: /opt/homebrew/opt/node@14/bin:/opt/homebrew/sbin:/opt/homebrew/bin:/usr/local/sbin:/usr/local/bin:./opt/homebrew/opt/node@14/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Users/qianyong/.fig/bin:/Users/qianyong/.local/bin:/Users/qianyong/apache/apache-maven-3.8.4/bin:/Users/qianyong/work/zuzu_jdk_1.8/bin:/Users/qianyong/person/go-workspace/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: ??? (???) - ???
Current terminal session: ???
Current working directory: ???
Current window identifier: ???</pre>
</p>
</details>

**Comment by yongqian0816** [comment]
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/61683111/180342427-740ca41b-19bd-4acb-bb3a-9cfd96299ff4.png">

**Comment by AngelaUncle** [comment]
me too

**Comment by wat-doing** [comment]
Yep, same here.  This seen when running "Connect Local Shell" in SecureCRT
